DEVELOPING_OPAL/demos/src/main/scala/org/opalj/ai/MethodsThatAlwaysReturnAPassedParameter.scala

```diff
@@ -22,61 +23,54 @@ object MethodsThatAlwaysReturnAPassedParameter extends ProjectAnalysisApplicatio
 
     override def title: String = "identify methods that always return a given parameter"
 
-    override def description: String = {
+    override def description: String =
         "identifies methods that either always throw an exception or return a given parameter"
-    }
 
     override def doAnalyze(
         theProject:    Project[URL],
         parameters:    Seq[String],
-        isInterrupted: () => Boolean
-    ): BasicReport = {
+        isInterrupted: () => Boolean): BasicReport = {
 
         val methods = for {
             classFile <- theProject.allClassFiles.par
-            method <- classFile.methods
+            method    <- classFile.methods
             if method.body.isDefined
             if method.descriptor.returnType.isReferenceType
             if (
                 method.descriptor.parametersCount > 0 &&
-                method.descriptor.parameterTypes.exists(_.isReferenceType)
+                    method.descriptor.parameterTypes.exists(_.isReferenceType)
```

Comment:
I don't mind this too much, but I could live without this additional indentation.

Reply:
I guess this can be changed with [this](https://scalameta.org/scalafmt/docs/configuration.html#indentoperator)